### ISSUE TITLE
Working AHK script for Odyssey

### ISSUE THREAD

**Issue by cowboy** [Numerlor/Auto_Neutron#55]
FYI the current AHK script doesn't work for me in Odyssey. My map looks like this (which is very different than the map in the README):

![ed odyssey galaxy map](https://github.com/user-attachments/assets/bb50d040-8b4b-4a79-928a-b0531fc62ccf)

This script is working for me (tested with AutoHotkey 1.1.37.02). Just uncheck "Simple mode", paste this in, and replace the two instances of `YourGalaxyMapKey` with your galaxy map key. [AHK key reference](https://www.autohotkey.com/docs/v1/lib/Send.htm#Parameters)

```ahk
; Auto_Neutron AHK script for Odyssey v1.0.0

SetKeyDelay, 50, 50
; open map
send, {YourGalaxyMapKey}
sleep, 850

; focus on search field
send, {Up}{Space}

; save previous clipboard contents
ClipOld := ClipboardAll

; system is the variable with the injected system
Clipboard := system

; paste clipboard
Send, ^v

; restore previous clipboard
Clipboard := ClipOld
ClipOld =

sleep, 1000
send, {enter}

; Screen click seems to be needed here
MouseMove, A_ScreenWidth/2, A_ScreenHeight/2
Click

; Hold space for a second to actually lock in destination
sendInput, {Space Down}
sleep, 1000
sendInput, {Space Up}
sleep, 100

; close map
send, {YourGalaxyMapKey}
```

**Comment by Numerlor** [comment]
Hi, would you mind opening a merge request? The current script is stored here https://github.com/Numerlor/Auto_Neutron/blob/master/auto_neutron/constants.py#L49-L67

I'll download the game and try it out. Then it'll go into the next release hopefully along with AHK 2 suppport

**Comment by Numerlor** [comment]
I had some "problems" previously with re-running as icons kept filling up the tray, I'll see how things will go; haven't really looked at anything yet apart from knowing AHK 2 is a thing. The plan is to keep support for both as 2's penetration doesn't seem that high

**Comment by cowboy** [comment]
From what I can tell, if you want to support both versions of AHK, you'll need 2 versions of the scripts, because the syntax isn't backwards compatible. If you're interested, I can create a v1 version of the above script, that uses the same approach.

`#SingleInstance Force` allows the script to be re-run without spawning a new instance, see [the docs](https://www.autohotkey.com/docs/v2/lib/_SingleInstance.htm). Works in v1 and v2.

**Comment by Numerlor** [comment]
Yeah 2 implementations is the plan. The current stdin piping seems to be working fine without launching new processes so I'm not too concerned about changing that behaviour

**Comment by cowboy** [comment]
Ok, good luck then

**Comment by cowboy** [comment]
One other thing I should point out: it seems that, generally, the first item in the dropdown list when searching in the galaxy map always matches the system name that was entered _unless_ you have any bookmarks that also match, in which case those will appear first. There doesn't seem to be any way around this. Just something to note.

**Comment by matthiasfabry** [comment]
Neither script works for me. the v2 doesn't even compile due to syntax errors, and the v1 version doesn't work because {up}{space} doesn't select the system input box. In fact, I cannot find any key combination that selects the Odyssey search box other than mouse.

Any ideas?
